sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -98,11 +97,13 @@ def on_request(self, request: PipelineRequest[HTTPRequestType]) -> None:
         self._enforce_https(request)
 
         if self._token is None or self._need_new_token:
-            if self._enable_cae:
-                self._token = self._credential.get_token(*self._scopes, enable_cae=self._enable_cae)
+            options: Dict[str, Any] = {"enable_cae": self._enable_cae} if self._enable_cae else {}
+            if isinstance(self._credential, ExpandedTokenCredential):
```

Comment:
Will opt to do a `hasattr` check for "get_expanded_token" since isinstance seems inconsistent in Python 3.8 (and this approach is probably more reliable overall)